packages/expo-doctor/src/checks/DirectPackageInstallCheck.ts

```diff
@@ -56,6 +55,33 @@ export const directPackageInstallCheckItems: DirectPackageInstallCheckItem[] = [
       )}`,
     sdkVersionRange: '>=49.0.0',
   },
+  {
+    packageName: '@expo/prebuild-config',
```

Comment:
I think this could also get the `This package should not be installed directly in your project.` warning, right?